File: jvm-experiments-commons/src/test/java/kr/kth/commons/period/TimeSpecTest.java
package kr.kth.commons.period;

import kr.kth.commons.AbstractTest;
import lombok.extern.slf4j.Slf4j;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

/**
 * {@link TimeSpec} 단위 테스트
 * User: sunghyouk.bae@gmail.com
 * Date: 12. 9. 30.
 */
@Slf4j
public class TimeSpecTest extends AbstractTest {

	@Test
	public void testTimeSpec() {
		assertEquals(0, TimeSpec.ZeroTimestamp.getTime());
	}
}

File: jvm-experiments-timeperiod/src/test/scala/kr/kth/timeperiod/TimeSpecTest.scala
package kr.kth.timeperiod

import kr.kth.commons.slf4j.Logging
import org.junit.{Assert, Test}

/**
 * {@link TimeSpec} 단위 테스트
 * User: sunghyouk.bae@gmail.com
 * Date: 13. 1. 13.
 */
class TimeSpecTest extends Logging {
  @Test
  def testTimeSpec() {
    Assert.assertEquals(0, TimeSpec.ZeroTick)
  }
}
